pkgs/development/python-modules/gymnasium/default.nix

```diff
@@ -60,6 +61,8 @@ buildPythonPackage rec {
     scipy
   ];
 
+  doCheck = !stdenv.isDarwin;
```

Comment:
Could you add a comment here stating why `doCheck` is set?
I think a copy paste of the PR body will do.